middle_end/flambda2/simplify/simplify_let_cont_expr.ml

```diff
@@ -1030,15 +1219,25 @@ let prepare_dacc_for_handlers dacc ~env_at_fork ~params ~is_recursive
     is_exn_handler,
     join_result.extra_params_and_args )
 
-let simplify_handler ~simplify_expr ~is_recursive ~is_exn_handler
-    ~invariant_params ~params cont dacc handler k =
+and simplify_handler ~simplify_expr ~is_recursive ~is_exn_handler ~lifted_params
+    ~invariant_params ~params ~all_extra_params cont dacc handler k =
+  let all_params = Bound_parameters.append invariant_params params in
```

Comment:
`all_params` don't contains the extra_params. I don't have a good idea for the name, but it's a bit confusing

Reply:
Maybe `all_normal_params`? Or we can just revert this part of the diff, now that `all_params` is only used once after refactorings.

Reply:
I've done what @Ekdohibs suggested